temporalcli/commandsgen/commands.yml

```
@@ -287,6 +288,81 @@ commands:
     option-sets:
       - workflow reference
 
+
+  - name: temporal activity update
+    summary: Update Activity options
+    description: |
+      Update Activity options. Specify the Activity and Workflow IDs, and options you want to update. 
+      Updates are incremental, only changing the specified options. 
+
+      ```
+      temporal activity update \
+          --activity-id YourActivityId \
+          --workflow-id YourWorkflowId \
+          --task-queue NewTaskQueueName \
+          --schedule-to-close-timeout DURATION\
+          --schedule-to-start-timeout DURATION\
+          --start-to-close-timeout DURATION\
+          --heartbeat-timeout DURATION\
+          --retry-initial-interval DURATION\
+          --retry-maximum-interval DURATION\
+          --retry-backoff-coefficient NewBackoffCoefficient\
+          --retry-maximum-attempts NewMaximumAttempts"
+      
+      ```
+    options:
+      - name: activity-id
+        type: string
+        description: Activity ID.
+        required: true
+      - name: task-queue
+        type: string
+        description: Name of the task queue for the Activity.
+      - name: schedule-to-close-timeout
+        type: duration
+        description: |
+          Indicates how long the caller is willing to wait for an activity completion. 
+          Limits how long retries will be attempted.
+      - name: schedule-to-start-timeout
+        type: duration
+        description: |
+          Limits time an activity task can stay in a task queue before a worker picks it up. 
+          This timeout is always non retryable, as all a retry would achieve is to put it back into the same
+          queue. Defaults to `schedule_to_close_timeout` or workflow execution timeout if not specified.
```

Avoid backticks unless you are intending to specify the metavariable for this option (see style guide).

```suggestion
          queue. Defaults to the schedule-to-close timeout or workflow execution timeout if not specified.
```